<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface_Contrat_RH</class>
 <widget class="QMainWindow" name="Interface_Contrat_RH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des contrats</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rssources.qrc">
    <normaloff>:/images/logo_erp.png</normaloff>:/images/logo_erp.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>460</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>641</width>
       <height>400</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Traiter les données</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_contrat_ajouter">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>360</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_contrat_id">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>74</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Contrat Id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_contrat_verifier">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>600</width>
        <height>226</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Données de contrat</string>
      </property>
      <widget class="QLabel" name="label_duree_contrat">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>70</y>
         <width>139</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Durée de contrat (mois)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_montant">
       <property name="geometry">
        <rect>
         <x>348</x>
         <y>70</y>
         <width>48</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Montant</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_date_fin">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>30</y>
         <width>64</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Date de fin</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_etat_contrat">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>110</y>
         <width>89</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Etat de contrat</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_type_contrat">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>92</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Type de contrat</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_indemnite">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>150</y>
         <width>130</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Indemnites de rupture</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_type_rupture">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>92</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Type de rupture</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_contrat_date_fin">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>30</y>
         <width>138</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_date_debut">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>84</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Date de début</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_contrat_date_debut">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_contrat_type">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>110</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Contrat à durée indéterminée</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Contrat de travail à temps partiel</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Contrat d'apprentissage</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Contrat à durée déterminée</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_contrat_rup">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>150</y>
         <width>120</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Démission</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Licenciement </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Décès</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Retraite</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_contrat_etat">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>110</y>
         <width>86</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>En cours</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Acheve</string>
        </property>
       </item>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_contrat_montant">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>70</y>
         <width>62</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="maximum">
        <double>100150.000000000000000</double>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="doubleSpinBox_contrat_indem">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>150</y>
         <width>62</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="maximum">
        <double>100150.000000000000000</double>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBox_contrat_duree">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>42</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
      </widget>
      <widget class="QLabel" name="label_statut">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>39</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Statut</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_contrat_statut">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>190</y>
         <width>99</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Payee</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Non payee</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_contrat_modifier">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>360</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_contrat_supprimer">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>360</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_cont_id">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_travailleur_id">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Travailleur Id</string>
      </property>
     </widget>
     <widget class="QFrame" name="logo_edf">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>20</y>
        <width>169</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/EdfErp.png);
image: url(:/images/logo_edf_erp.png);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_id_emp">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>89</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>98</width>
       <height>28</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Afficher les données</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_contrat_afficher">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>75</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Actualiser</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_contrat">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>621</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_cont_id</tabstop>
  <tabstop>lineEdit_contrat_date_debut</tabstop>
  <tabstop>spinBox_contrat_duree</tabstop>
  <tabstop>comboBox_contrat_type</tabstop>
  <tabstop>comboBox_contrat_rup</tabstop>
  <tabstop>lineEdit_contrat_date_fin</tabstop>
  <tabstop>doubleSpinBox_contrat_montant</tabstop>
  <tabstop>comboBox_contrat_etat</tabstop>
  <tabstop>doubleSpinBox_contrat_indem</tabstop>
  <tabstop>pushButton_contrat_ajouter</tabstop>
  <tabstop>pushButton_contrat_modifier</tabstop>
  <tabstop>pushButton_contrat_supprimer</tabstop>
  <tabstop>pushButton_contrat_verifier</tabstop>
  <tabstop>pushButton_contrat_afficher</tabstop>
  <tabstop>tableView_contrat</tabstop>
 </tabstops>
 <resources>
  <include location="rssources.qrc"/>
 </resources>
 <connections/>
</ui>
